themes/black-purple-2025/layouts/_partials/header.html

```
@@ -0,0 +1,19 @@
+<header class="sticky top-0 z-10 h-16 bg-neutral-200/50 dark:bg-neutral-900/50 backdrop-blur-xl">
+  <div class="container mx-auto flex h-full items-center justify-between flex-col sm:flex-row">
+    <a class="hover:text-accent font-bold text-lg" href="{{ relLangURL "/" }}">Carlos A. Gomes</a>
+
+    <aside class="flex gap-4">
+      {{ partial "theme-toggle.html" . }}
+
+      {{ partial "language-switcher.html" . }}
+
+      <nav>
+        <ul class="flex gap-4">
+          <li><a class="hover:text-accent" href="{{ relLangURL "/" }}">{{ i18n "home" }}</a></li>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)